### ISSUE TITLE
ACT bc4a75: fix `aria_child_valid` WCAG mapping from 4.1.2 to 1.3.1

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1957]
`aria_child_valid` should not map to WCAG 4.1.2, but to 1.3.1 for ACT ruleID `bc4a75`

- [x] also change the Help to list 1.3.1, not 4.1.2

The ACT rule maps to WCAG requirement:
1.3.1: Info and Relationships (Level A)

### Supporting notes and details
 [ACT Master 2024.xlsx](https://ibm.box.com/s/evec4kfixgzjw8tmghnr9f13xxw99e33)

**Comment by philljenkins** [comment]
mapping change coded in phill-ACT-mapping
changed 4.1.2 to 1.3.1 in both rule and Help reference

**Comment by philljenkins** [comment]
Help formats correctly
and references 1.3.1
<img width="1726" alt="Screenshot 2024-07-26 at 12 45 37 PM" src="https://github.com/user-attachments/assets/611c5087-1de3-4067-9a40-fc81ecc5c7c6">

**Comment by philljenkins** [comment]
correct in Rule set

<img width="924" alt="Screenshot 2024-07-26 at 1 00 21 PM" src="https://github.com/user-attachments/assets/942b0472-27f5-4c4d-927e-4dc3626ea50e">

**Comment by philljenkins** [comment]
Reports correctly against ACT test cases `bc4a75` in [act-report-v2 2024.txt](https://ibm.box.com/s/or6i901ke3etyv5xoo5e61yynx6xem77)

> ARIA required owned elements (https://www.w3.org/WAI/standards-guidelines/act/rules/bc4a75)
>  + Passed Example 1: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/5c4aa70ce778897a8f9601621075c0f0c9abdc65.html
>  + Passed Example 2: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/854ec3db8780fe5be957fc1c29f279a3e5fc7969.html
>  + Passed Example 3: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/12a2da38812d7bf356f0092674c1c21802faf30d.html
>  + Passed Example 4: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/e74d875a66842a61c1667ec316b9d455e7e3a331.html
>  + Passed Example 5: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/a1826280426b6a76f0c871084146983b6f0faa9b.html
>  + Passed Example 6: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/51307c16b15d283b9f6ba7e4edc1bb99449f3f37.html
>  + Failed Example 1: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/dd4d60acdda2a92253d4fc09cff248e9e0e3eb74.html
>  + Failed Example 2: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/0763ce51664b522eb3ed2c5479e11f4ed91e871c.html
>  + Failed Example 3: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/0fd4574e8dd585f4cb14c20f9966bf94f2139ea9.html
>  + Failed Example 4: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/874032cb82216878366f02dd2d98e6c8047a1612.html
>  + Failed Example 5: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/f656ec33b2faf9fa804c61d09102fc70e1b916d2.html
>  + Failed Example 6: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/5e0e88f9ed776c89735d7db606c1381a7a1fb877.html
>  + Failed Example 7: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/52c725e462af074a3559cf4bf4d4dd2386168938.html
>  + Inapplicable Example 1: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/83d80bc34891ae13f05150c8c677028591f1d199.html
>  + Inapplicable Example 2: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/9ed4f5f7c0a9b8377e7652022430cbd5b1eddccf.html
>  + Inapplicable Example 3: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/a700af25effa2ba2bfad2429705e634fcd0142d6.html
>  + Inapplicable Example 4: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc4a75/837f92d0ac41c14e55782991cbab75975b492702.html
